src/lib/api/project/compare-branches.ts

```diff
@@ -0,0 +1,34 @@
+import debug = require("debug");
+import { requestsManager } from "snyk-request-manager";
+import { updateProject } from ".";
+import { CompareAndUpdateBranchesResponse } from "../../types";
+
+export async function compareAndUpdateBranches(
```

Comment:
this needs to live somewhere else else `src/lib/api/*` is only for the API call implementation.

This is now "business" logic specific to sync. I think `src/lib/project` is a better place